test/typescript-karma/Data/Formula/Functions/AVERAGEA.test.js

```diff
@@ -0,0 +1,16 @@
+import DataTable from '/base/code/es-modules/Data/DataTable.js';
+import Formula from '/base/code/es-modules/Data/Formula/Formula.js';
+
+
+QUnit.test('Formula.processorFunctions.AVERAGEA', function (assert) {
+    const table = new DataTable({
+            columns: { values: [1, 3, 4, 5, 6, 7, false, true, null, '7'] }
```

Comment:
What am I missing here?
![image](https://user-images.githubusercontent.com/57004120/235093796-d6032ba3-e8e8-41b3-859b-0b9b9beae38e.png)

Reply:
FALSE and NULL get skipped. Strings and TRUE are counted as `1`.